src/hooks/useBalance.ts

```diff
@@ -40,7 +37,7 @@ export const useBalance = <TAddressMap extends AddressMap = AddressMap>(tokenAdd
   const results = useQueries(
     networkIds.map(networkId => ({
       queryKey: balanceQueryKey(address, tokenAddressMap, networkId),
-      enabled: !!address && (isTestMode ? isTestnet(networkId) : !isTestnet(networkId)),
+      enabled: !!address,
```

Comment:
why is this being removed?

Reply:
it breaks useGohmTokemakBalance() when you're connected to a testnet which leaves Total Staked Balance on the stake page perpetually stuck as a Skeleton. That seemed like incorrect behavior, but I can re-add if you'd like. Better behavior than either of these is just returning 0 when `(isTestMode ? isTestnet(networkId) : !isTestnet(networkId))` computes as false.

<img width="818" alt="image" src="https://user-images.githubusercontent.com/92545857/175618667-a482130b-2c09-44c8-b1e4-2cda0ff8616d.png">

Reply:
oh nice